styleguide/derek/examples/_office365.html

```diff
@@ -2151,4 +2151,59 @@ <h4 class="center">The best part about all of our plans?<br/>
       </div>
     </div>
   </div>
-</div>
\ No newline at end of file
+
+  <!-- Additional Resources -->
+
+  <div class="container-fluid">
+    <div class="container additional-resources-container">
```

Comment:
Do we need this class? "additional-resources-container"? Does it have any effect if removed?